source/4_tools/regina/src/regina.cpp

```diff
@@ -40,8 +40,14 @@ namespace rex
     app_params.engine_params.app_update_func   = update;
     app_params.engine_params.app_shutdown_func = shutdown;
 
-    app_params.create_window = true;
-
     return app_params;
   }
+
+#ifndef REX_ENABLE_AUTO_TESTS
+
+  ApplicationCreationParams app_entry(PlatformCreationParams&& platformParams)
+  {
+    return create_regina_app_creation_params(rsl::move(platformParams));
+  }
+#endif
```

Comment:
We should come up with a solution to get rid of this macro

Reply:
Regina can have a start project which checks at runtime if autotests are specified and then calls into Regina or ReginaTest, then you don't need a macro, you do need an extra project though, one that depends on both Regina and ReginaTest